src/style.css

```diff
@@ -1 +1,93 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+@media screen and (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+@media screen and (max-width: 1200px) {
+  .header {
+    width: 1200px;
+  }
+}
+
+.nav {
+  display: block;
+}
+
+.links {
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  position: relative;
+  text-transform: uppercase;
+  color: #000;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.n_list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  --main-color: #00acdc;
+}
+
+.logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  text-align: center;
+  padding: 0;
+}
+
+.item {
+  margin-right: 20px;
+}
+
+.links:hover {
+  color: var(--main-color);
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  width: 37px;
```

Comment:
Try changing the width so that the line is always the full width of any link
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/f6273eb6-7f9e-4ce8-b095-bed9b9d5da24)